File: generate_database.ts
import fs from 'fs';
import uuid from 'uuid';
import data from './app/assets/orientation-bacc.json';

var assetsPath = './app/assets';
// Find all bacc series
var new_data = [];
for (let content of data) {
  new_data.push({id: uuid()});
  for (let [key, value] of Object.entries(content)) {
    new_data.push(key: value);
  }
}
console.log(new_data);
const result = {updated_at: new Date(), data};
const jsonContent = JSON.stringify(result);
const outFile = '/faculty.json';
fs.writeFile(assetsPath + outFile, jsonContent, 'utf8', function(err) {
  if (err) {
    console.log('An error occured while writing JSON Object to File.');
    console.log(err);
  } else {
    console.log('./assets/faculty.json generated successfully!');
  }
});

File: generate_database.py
import json
import uuid
from datetime import datetime

#Open array file
file = './app/assets/orientation-bacc.json'
with open(file, encoding='utf8') as json_file:
  data = json.load(json_file)
  for p in data:
    print(p)
    p['id'] = str(uuid.uuid1())

with open('./app/assets/database.json', 'w', encoding='utf8') as outfile:
  json.dump({'updated_at': str(datetime.now().isoformat()), 'data': data}, outfile, ensure_ascii=False)
